vrealize_operations_integration_sdk/validation/adapter_definition_validator.py

```diff
@@ -0,0 +1,36 @@
+#  Copyright 2022 VMware, Inc.
+#  SPDX-License-Identifier: Apache-2.0
+
+import json
+import logging
+import os
+from json import JSONDecodeError
+
+from vrealize_operations_integration_sdk.describe import json_to_xml
+from vrealize_operations_integration_sdk.logging_format import PTKHandler, CustomFormatter
+from vrealize_operations_integration_sdk.validation.describe_checks import validate_describe
+from vrealize_operations_integration_sdk.validation.result import Result
+
+logger = logging.getLogger(__name__)
+logger.setLevel(os.getenv("LOG_LEVEL", "INFO").upper())
+consoleHandler = PTKHandler()
+consoleHandler.setFormatter(CustomFormatter())
+logger.addHandler(consoleHandler)
+
+
+def validate_adapter_definition(project, request, response):
+    result = Result()
+    if response.status_code == 204:
+        result.with_error("No adapter description was returned by the adapter.")
```

Comment:
This will only get displayed if they specifically run the adapter_definition endpoint and it doesn't return anything. I think error is appropriate, but maybe I can tweak the message a bit.

Reply:
After further thought, I reduced the severity to warning and added some additional context